src/test/java/com/wikia/webdriver/testcases/flagTests/flagTest.java

```diff
@@ -0,0 +1,71 @@
+package com.wikia.webdriver.testcases.flagTests;
+
+import com.wikia.webdriver.common.properties.Credentials;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+/**
+ * Created by Mariusz on 2015-06-08.
+ */
+public class flagTest extends NewTestTemplate {
+    Credentials credentials = config.getCredentials();
+
+
+    @Test(groups = {"addFlag_01"})
+
+    public void addFlag_01() {
+
+        List list = new ArrayList();
+        WikiBasePageObject base = new WikiBasePageObject(driver);
+
+        base.logInCookie(credentials.userNameStaff, credentials.passwordStaff, wikiURL);
+
+        //  WikiBasePageObject base = new WikiBasePageObject(driver);
+        wikiURL = "http://preview.runescape-test.wikia.com/wiki/Special:Random";
```

Comment:
This URL should be declared in an external file such as URLsContent in contentpatterns package or perhaps it might be simpler to use the clickRandom method